src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -5920,6 +5925,886 @@ class quest_resizing_the_situation_horde : public quest_resizing_the_situation
+        me->SetUnitFlag(UNIT_FLAG_PLAYER_CONTROLLED | UNIT_FLAG_IMMUNE_TO_PC);
+    }
+
+    void JustAppeared() override
+    {
+        Unit* owner = me->GetOwner();
+
+        Player* player = owner->ToPlayer();
+        if (!player)
+            return;
+
+        Creature* shuja = FindCreatureIgnorePhase(me, "shuja_grimaxe_ogre_ruins_prisoner");
+        if (!shuja)
+            return;
+
+        Conversation* conversation = Conversation::CreateConversation(CONVERSATION_WARLORD_GRIMAXE_SPAWN_Q59942, player, *player, player->GetGUID(), nullptr, false);
+        if (!conversation)
+            return;
+
+        conversation->AddActor(ACTOR_SHUJA_GRIMAXE_PRISONER, 0, shuja->GetGUID());
+        conversation->AddActor(ACTOR_WARLORD_GRIMAXE_Q59942, 1, me->GetGUID());
+        conversation->Start();
+
+        me->GetMotionMaster()->MoveFollow(player, 0.0f, float(M_PI / 4.0f));
+        _events.ScheduleEvent(EVENT_CAPTAIN_GARRICK_RIDE_BOAR_CHECK_OWNER, 1s);
+    }
+
+    void DamageTaken(Unit* /*attacker*/, uint32& damage, DamageEffectType /*damageType*/, SpellInfo const* /*spellInfo = nullptr*/) override
+    {
+        // *** HACK ***
+        damage = 1;
+    }
+
+    void SpellHit(WorldObject* /*caster*/, SpellInfo const* spellInfo) override
+    {
+        switch (spellInfo->Id)
+        {
+            case SPELL_PING_GARRICK_TORGOK:
+            {
+                Creature* shuja = FindCreatureIgnorePhase(me, "shuja_grimaxe_ogre_ruins_prisoner");
+                if (!shuja)
+                    break;
+
+                Player* player = me->GetOwner()->ToPlayer();
+                if (!player)
+                    break;
+
+                Creature* shujaPersonal = shuja->SummonPersonalClone(shuja->GetPosition(), TEMPSUMMON_MANUAL_DESPAWN, 0s, 0, 0, player);
+                if (!shujaPersonal)
+                    break;
+
+                me->SetReactState(REACT_AGGRESSIVE);
+                _shujaGUID = shujaPersonal->GetGUID();
+                _events.ScheduleEvent(EVENT_CAPTAIN_GARRICK_RIDE_BOAR_TALK_TO_HENRY, 1s);
+                break;
+            }
+            default:
+                break;
+        }
+    }
+
+    void UpdateAI(uint32 diff) override
+    {
+        _events.Update(diff);
+
+        while (uint32 eventId = _events.ExecuteEvent())
+        {
+            switch (eventId)
+            {
+                case EVENT_CAPTAIN_GARRICK_RIDE_BOAR_CHECK_OWNER:
+                {
+                    if (Unit* owner = me->GetOwner())
+                    {
+                        if (Player* player = owner->ToPlayer())
+                        {
+                            if (player->GetQuestStatus(QUEST_RE_DEATHER) == QUEST_STATUS_INCOMPLETE)
+                            {
+                                if (player->IsInCombat() && !me->IsInCombat())
+                                    me->Attack(player->GetVictim(), true);
```

Comment:
AI()->AttackStart() directly sets chase movement, which is something we might want here
```suggestion
                                    me->AI()->AttackStart(player->GetVictim());
```